en/admin-guide/pgadmin/usage.rst

```diff
@@ -6,7 +6,7 @@ Using Pgadmin
 SQL query
 ---------
 
-``.. figure:: images/001_roles.jpeg``
+ .. figure:: images/001_roles.jpeg
```

Comment:
I already fixed this in https://github.com/QGEP/docs/pull/18
This means that whichever pull gets merged first would automatically require changes in the other one in order to be cleared for merge. Can you rebase without this?